features/step_definitions/assertion_steps.rb

```diff
@@ -1,6 +1,10 @@
 Then(/^I should see an image named "([^"]*)"$/) do |image_name|
+  image = Image.find_by(name: image_name)
   within '#images' do
```

Comment:
I wanted to make sure both conditions are met. Could probably do it on one line but I would settle for this implementation.